libs/ledger-live-common/.unimportedrc.json

```diff
@@ -290,6 +290,7 @@
     "**/datasets/**",
     "**/mock/fixtures/**",
     "src/deviceSDK/**",
+    "src/config/sharedConfig.ts",
```

Comment:
Is it actually intended ? Maybe it's because we import this on LLD and LLM only

Reply:
yes, as we import this on LLD and LLM.
I found that if I don't add this file here. There would be an CI error.